src/components/Project/news/index.jsx

```diff
@@ -1,101 +1,97 @@
-                平日・夕方までの勤務!伊勢市の伊勢赤 十字病院で医療事務求人
-              </span>
-              <div className={styles['card']}>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>給与</span>
-                  <span className={styles['text']}>月給148,000円〜</span>
-                </div>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>所在地</span>
-                  <span className={styles['text']}>三重県伊勢市</span>
-                </div>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>最寄駅</span>
-                  <span className={styles['text']}>伊勢市駅 徒歩15分</span>
-                </div>
-              </div>
-            </div>
-            <div className={styles['card-item']}>
-              <div className={styles['woman3']}></div>
-              <span className={styles['subtext-title']}>
-                平日・夕方までの勤務!伊勢市の伊勢赤 十字病院で医療事務求人
-              </span>
-              <div className={styles['card']}>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>給与</span>
-                  <span className={styles['text']}>月給148,000円〜</span>
-                </div>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>所在地</span>
-                  <span className={styles['text']}>三重県伊勢市</span>
-                </div>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>最寄駅</span>
-                  <span className={styles['text']}>伊勢市駅 徒歩15分</span>
-                </div>
-              </div>
-            </div>
-            <div className={styles['card-item']}>
-              <div className={styles['woman4']}></div>
-              <span className={styles['subtext-title']}>
-                平日・夕方までの勤務!伊勢市の伊勢赤 十字病院で医療事務求人
-              </span>
-              <div className={styles['card']}>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>給与</span>
-                  <span className={styles['text']}>月給148,000円〜</span>
-                </div>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>所在地</span>
-                  <span className={styles['text']}>三重県伊勢市</span>
-                </div>
-                <div className={styles['text-list']}>
-                  <span className={styles['label']}>最寄駅</span>
-                  <span className={styles['text']}>伊勢市駅 徒歩15分</span>
-                </div>
-              </div>
-            </div>
-            <div
-              className={`${styles['slide-button']} ${styles['right']}`}
-            ></div>
-          </div>
+          <div
+            className={`${styles['slide-button']} ${styles['right']}`}
+            ref={nextRef}
+          ></div>
```

Comment:
空要素を実装するのであれば、開始タグの最後に/をつけるだけでOKです！
```suggestion
          />
```

Reply:
確認ありがとうございます。

空要素を開始タグの最後に/をつける修正をいたしましたのでご確認お願いします。

[3a8d97a](https://github.com/gizumo-education/front-coding/pull/219/commits/3a8d97ac5d3f2a576be30ebd014c094003e4d114)